hack/addons/update-aws-ebs-csi.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+set -euo pipefail
```

Comment:
Please update this to match https://github.com/d2iq-labs/capi-runtime-extensions/pull/207 🙏